Source/Core/AST/AbsyCmd.cs

```diff
@@ -4003,6 +4015,9 @@ protected override Cmd ComputeDesugaring(PrintOptions options)
         Contract.Assert(e != null);
         Expr copy = Substituter.ApplyReplacingOldExprs(calleeSubstitution, calleeSubstitutionOld, e.Condition);
         AssumeCmd assume = new AssumeCmd(this.tok, copy);
+        if (idAttr is not null) {
```

Comment:
The `id` of the call statement itself is included in the `id` for the postcondition to avoid exactly that duplication.

It's useful to know whether a postcondition (or a precondition) from a particular call is used, rather than whether the postcondition is used in general. Some of the things that it can help you identify:
* a context-dependent contradictory postcondition (and, in particular, what parts of it lead to a contradiction)
* a postcondition that's only rarely used, and perhaps better established in other ways (such as by splitting a lemma into multiple lemmas)

When combined with split-on-every-assert, this can help you identify exactly how each lemma you use contributes to each goal you're trying to prove.

Reply:
> The id of the call statement itself is included in the id for the postcondition to avoid exactly that duplication.

I see! Could you rename `id` to `callId` ?

> Can you add a test-case where there is more than 1 call to a callee, and the coverage results are different per call?

Consider still doing this.